Governance/publish/publish-options.md

```diff
@@ -2,41 +2,55 @@
 Once the data is created and ready for publication, **where** can it be published? 
 
 One of the main factors is what services are available, this is largely determined by the researcher's institution and/or organization.
-In some cases, there might be the option of contributing to a large-scale project that has its own publishing procedure established.
+Another important consideration is the kind of data, depending on format, size etc. using a public repository might be suitable and at time easier.
+In some cases, depending on the dataset, using a public repository is suitable and often the easiest option. ori
+On the other end of the scale, some data could be produced on purpouse or simply be suitable to contribute to a large-scale project that has its own publishing procedure established.
 Here we are covering the most common use cases for our community.
 
-## BoM
+::::{tab-set}
+:::{tab-item} BoM 
+Insert here pathways for BoM researchers...

```

Comment:
Request review/contribution from someone at BoM - who?

Reply:
I think Avi was happy to contribute and Griffith (?) mentioned they're publishing with C3S so he could contribute that